index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <script src="https://kit.fontawesome.com/c373984c3f.js" crossorigin="anonymous"></script>
+        <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,900;1,800&family=Roboto:wght@100;300;400;500;700;900&display=swap" rel="stylesheet">
+        <link rel="stylesheet" href="style.css">
+        <title>My Portfolio</title>
+    </head>
+
+    <body>
+        <header class="intro">
+
+            <div class="toolbar">
+
+                    <a href="#" class="toolbar-logo font-family-roboto">Belden Mugauri</a>
+  
+                    <nav id="nav-bar" data-visible="false" > 
+
+                        <button class="menu-button" id="menu-btn" aria-controls="nav-bar">
+                            <i class="fa-solid fa-bars"><span class="sr-only">Menu</span></i>
+                        </button>
+                            
+                        <ul class="primary-navigation font-family-roboto">
+                            <li class="nav-item font-family-roboto">
+                                <a href="#">Portfolio</a>
+                            </li>
+                
+                            <li class="nav-item font-family-roboto">
+                                <a href="#">About</a>
+                            </li>
+                
+                            <li class="nav-item font-family-roboto"> 
+                                <a href="#">Contact</a> 
+                            </li>
+                   </ul>
```

Comment:
:orange_circle: [OPTIONAL] Make sure to hide this section on the mobile version, as it causes the page to overflow from the normal flow. Another option is to remove it; there is a special activity for toggling the menu.

![fix-this](https://user-images.githubusercontent.com/26736582/212352934-7c5dbe5f-393d-40e7-94dc-2a7d644dde81.gif)